frontends/web/src/routes/buy/info.tsx

```diff
@@ -98,9 +98,18 @@ class BuyInfo extends Component<Props, State> {
         setConfig({ frontend: { skipBuyDisclaimer: e.target.checked }});
     }
 
+    private getCryptoName = (): string => {
+        const { accounts, code, t } = this.props;
+        if (!code) {
+            const onlyBitcoin = accounts.every((({ coinCode }) => isBitcoinOnly(coinCode)));
+            return onlyBitcoin ? 'Bitcoin' : t('buy.info.crypto');
+        }
+        const account = accounts.find(account => account.code === code);
+        return isBitcoinOnly(account!.coinCode) ? 'Bitcoin' : t('buy.info.crypto');
```

Comment:
`account!.coinCode` could be undefined, but `isBitcoinOnly` does not take undefined? Maybe better to catch the undefined case before separately.

Reply:
Find could return `undefined` if it would not find the account, but this should never happen. Reason `code` is only set if the user clicked on "Buy" inside the account, so the account should be there.

Added a check `if (account) {` anyways, to make 100% sure we have an account here and removed the `!` and added a fallback to `buy.info.crypto`